testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -60,7 +60,11 @@ impl ClusterBuilder {
         }
     }
 
-    pub async fn setup_cluster(&self, params: &ClusterBuilderParams) -> Result<Cluster> {
+    pub async fn setup_cluster(
+        &self,
+        params: &ClusterBuilderParams,
+        reset: bool,
```

Comment:
reset is a bit confusing name here

One would expect that when reset=true, setup_cluster would do more things, while it in fact do less(do not reset asg).

Can you rename it to smth more explicit like `preserve_nodes`?

Reply:
Sure thing. I saw that there's new change that added ClusterSwarm::clean_data, so I'll go ahead and name this `clean_data: bool`